core/src/test/java/org/apache/calcite/test/ReflectiveSchemaTest.java

```diff
@@ -953,13 +974,13 @@ public IntAndString(int id, String value) {
             false, (byte) 0, (char) 0, (short) 0, 0, 0L, 0F, 0D,
             false, (byte) 0, (char) 0, (short) 0, 0, 0L, 0F, 0D,
             new java.sql.Date(0), new Time(0), new Timestamp(0),
-            new Date(0), "1"),
+            new Date(0), "1", BigDecimal.valueOf(-1)),
         new EveryType(
             true, Byte.MAX_VALUE, Character.MAX_VALUE, Short.MAX_VALUE,
             Integer.MAX_VALUE, Long.MAX_VALUE, Float.MAX_VALUE,
             Double.MAX_VALUE,
             null, null, null, null, null, null, null, null,
-            null, null, null, null, null),
+            null, null, null, null, null, new BigDecimal("1.2")),
```

Comment:
I suggest to change `new BigDecimal("1.2"))` to `null` here.

Reply:
Yes, we should consider the null corner case